source/views/stoprint/components/notice.js

```diff
@@ -76,6 +83,7 @@ export class StoPrintNoticeView extends React.PureComponent<Props, State> {
 					style={styles.notice}
 					text={text}
 				/>
+				<Text style={styles.description}>{description}</Text>
```

Comment:
Shoot, I missed an important part of not displaying the `<Text />` element if no `description` was provided.

Reply:
ahh true